doc-src/ch-run_desc.rst

```diff
@@ -184,6 +207,127 @@ are tweaked. This section lists those changes.
     * `Fedora <https://fedoraproject.org/wiki/Features/UsrMove>`_
     * `Debian <https://wiki.debian.org/UsrMerge>`_
 
+Setting environment variables with :code:`--set-env-file`
+---------------------------------------------------------
+
+The purpose of :code:`--set-env-file[=FILE]` is to set environment variables
+originating by mechanisms such as Dockerfile :code:`ENV` statements or the
+Bash :code:`set` command.
+
+:code:`FILE` is a guest path. If not specified, it defaults to **FIXME**.
+
+The syntax of :code:`FILE` is key-value pairs separated by the first equals
+character (:code:`=`, ASCII 61), with optional single straight quotes
+(:code:`'`, ASCII 39) around the value stripped. No variable expansion,
+comments, etc. are provided. The value may be empty, but not the key. (This
+syntax is designed to accept the output of **FIXME: DOCKER SOMETHING** and
+shell :code:`set` or :code:`env`.) Examples of valid lines:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Line
+     - Key
+     - Value
+   * - :code:`FOO=bar`
+     - :code:`FOO`
+     - :code:`bar`
+   * - :code:`FLAGS=-march=foo -mtune=bar`
+     - :code:`FLAGS`
+     - :code:`-march=foo -mtune=bar`
+   * - :code:`FLAGS='-march=foo -mtune=bar'`
+     - :code:`FLAGS`
+     - :code:`-march=foo -mtune=bar`
+   * - :code:`FOO=`
+     - :code:`FOO`
+     - (empty string)
+   * - :code:`FOO=''`
+     - :code:`FOO`
+     - (empty string)
+   * - :code:`FOO=''''`
+     - :code:`FOO`
+     - :code:`''` (two single quotes)
```

Comment:
I don't think we should allow `FOO` or `''`as valid. They are not a key-value pair separated by `=`.

Reply:
Looks like I misread the doc syntax. Resolving.